experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -121,7 +121,7 @@ describe("FormAudience", () => {
     );
 
     expect(screen.getByTestId("isSticky")).toBeChecked();
-    expect(screen.getByTestId("isFirstRun")).toBeChecked();
+    expect(screen.queryByText("isFirstRun")).not.toBeInTheDocument();
```

Comment:
text will not be `isFirstRun`- it should be `First Run Experiment` or query by id

Reply:
@yashikakhurana as i told above querying by id results in error so i would rather change the text inside `queryByText` as per your mentions

Reply:
@yashikakhurana i have the made changes have a look at it